rs/tests/driver/src/k8s/tnet.rs

```diff
@@ -360,15 +361,23 @@ impl TNet {
             }
         );
 
+        // We make reservation for 42 vcpus only if VM uses 64 vcpus because there are
+        // already other k8s resources having resource requests that prevents reservation to succeeds.
+        // Note that VM still gets 64 vcpus.
+        let vcpus = min(36, vm_req.vcpus.get()).to_string();
```

Comment:
If we can't reserve more than 42 vCPUs shouldn't we have:
```suggestion
        let vcpus = max(42, vm_req.vcpus.get()).to_string();
```

Reply:
I think I mixed up min and max 🙈